lib/galaxy_test/api/test_dataset_collections.py

```diff
@@ -356,6 +357,88 @@ def test_collection_contents_limit_offset(self):
         assert len(offset_contents) == 1
         assert offset_contents[0]['element_index'] == 1
 
+    def test_get_suitable_converters_single_datatype(self):
+        response = self.dataset_collection_populator.upload_collection(self.history_id, "list:paired", elements=[
+            {
+                "name": "test0",
+                "elements": [
+                    {"src": "pasted", "paste_content": "123\n", "name": "forward", "ext": "bed"},
+                    {"src": "pasted", "paste_content": "456\n", "name": "reverse", "ext": "bed"},
+                ]
+            },
+            {
+                "name": "test1",
+                "elements": [
+                    {"src": "pasted", "paste_content": "789\n", "name": "forward", "ext": "bed"},
+                    {"src": "pasted", "paste_content": "0ab\n", "name": "reverse", "ext": "bed"},
+                ]
+            }
+        ])
+        self._assert_status_code_is(response, 200)
+        hdca_list_id = response.json()["outputs"][0]["id"]
+        converters = self._get("dataset_collections/" + hdca_list_id + "/suitable_converters")
+        expected = Counter([
```

Comment:
Why use Counter here ? If you want to compare the length of a list you can use something like assert len(converters.json()) == len(your_list_here]), or if you want to confirm that the lists are identical `converters.json() == your_list_here` (or if they're in random order throw in a `sorted(converters.json()) == sorted(your_list_here)`)